cpu/native/include/periph_conf.h

```diff
@@ -72,6 +72,15 @@
 #endif
 /** @} */
 
+/**
+ * @brief QDEC configuration
+ * @{
```

Comment:
You can remove this line: no need for a group with `@brief`

Reply:
It should then be done also for other @brief in file ?
```
/**
 * @brief UART configuration
 * @{
 */
```